src/main/java/org/mule/extension/file/common/api/util/UriUtils.java

```diff
@@ -50,7 +51,7 @@ public static URI createUri(String basePath, String filePath) {
     String fullPath;
     try {
       if (filePath.length() > 0) {
-        if (isAbsolute(filePath)) {
+        if (isAbsolute(filePath) || basePath.isEmpty()) {
```

Comment:
what will happen if the filePath is not an absolute path and the basePath is empty? will the URI constructor throw an exception?